docs/user-guide/measure.md

```diff
@@ -75,3 +73,43 @@ Once a measure is drawn, it is possible to add it as an [Annotation](annotations
 <img src="../img/measure/add-as-annotation-ex.jpg" class="ms-docimage"/>
 
 From this step the creation process is the same described in the [Annotations section](annotations.md#add-new-annotation).
+
+## Measurement on the 3D navigation
+
+Thanks to the new improvements made to the **Measure tool**, when [3D Navigation](side-bar.md#3d-navigation) is enabled, the editor has the ability to perform distance, area, point coordinates, height from terrain, angle and slope measurements on the 3D map.
+
+### Measure distance on the 3D navigation
+
+As soon as the measure window opens, by default, the **Measure distance in 3D space** option is selected <img src="../img/button/measure-distance-on-3d.jpg" class="ms-docbutton"/>. In order to perform a distance measure, each click on the map correspond to a segment of the line (at least one segment is needed) while the double click inserts the last line segment and ends the drawing session.
```

Comment:
As soon as the measure window opens, by default the **Measure distance in 3D space** option is selected <img src="../img/button/measure-distance-on-3d.jpg" class="ms-docbutton"/>. In order to perform a distance measure, each click on the map correspond to a segment of the line (at least one segment is needed) while the double click inserts the last line segment and ends the drawing session.